server/views/pages/sentence.njk

```diff
@@ -239,10 +239,10 @@
     {% set history %}
         {% set previousOrders %}
             {% if sentenceDetails.probationHistory.numberOfTerminatedEvents == 1 %}
-              <a href="/case/{{ crn }}/sentence/previous-orders">1 previous order</a><br />
+              <a href="/case/sentence/{{ crn }}/previous-orders">1 previous order</a><br />
```

Comment:
I think the route should still be `/case/{{ crn }}/sentence/previous-orders`

Reply:
b/e api matches /case/sentence/{{ crn }}/previous-orders.  can we pick this up on a later refactor

Reply:
f/e change done.  will most likely leave the b/e